web3deployment/KatalystDeploy/compileContractsV5.js

```diff
@@ -1,94 +0,0 @@
-            'IKyberNetworkProxy.sol' : {content: fs.readFileSync(contractPath + 'IKyberNetworkProxy.sol', 'utf8')},
-            'IKyberReserve.sol' : {content: fs.readFileSync(contractPath + 'IKyberReserve.sol', 'utf8')},
-            'IKyberTradeLogic.sol' : {content: fs.readFileSync(contractPath + 'IKyberTradeLogic.sol', 'utf8')},
-            'ISimpleKyberProxy.sol' : {content: fs.readFileSync(contractPath + 'ISimpleKyberProxy.sol', 'utf8')},
-            'KyberNetwork.sol' : {content: fs.readFileSync(contractPath + 'KyberNetwork.sol', 'utf8')},
-            'KyberNetworkProxy.sol' : {content: fs.readFileSync(contractPath + 'KyberNetworkProxy.sol', 'utf8')},
-            'KyberTradeLogic.sol' : {content: fs.readFileSync(contractPath + 'KyberTradeLogic.sol', 'utf8')}, 
-            'KyberHintHandler.sol' : {content: fs.readFileSync(contractPath + 'KyberHintHandler.sol', 'utf8')},
-            'ReentrancyGuard.sol' : {content: fs.readFileSync(contractPath + 'ReentrancyGuard.sol', 'utf8')},
-            'UtilsV5.sol' : {content: fs.readFileSync(contractPath + 'UtilsV5.sol', 'utf8')},
-            'WithdrawableV5.sol' : {content: fs.readFileSync(contractPath + 'WithdrawableV5.sol', 'utf8')} 
-        },
-        settings: {
-            outputSelection: { // return everything
-                '*': {
-                    '*': ['*']
-                }
-            },
-            // Optional: Optimizer settings
-            'optimizer': {
-                'enabled': true,
-                // Optimize for how many times you intend to run the code.
-                // Lower values will optimize more for initial deployment cost, higher
-                // values will optimize more for high-frequency usage.
-                'runs': 200,
-            },
-            'evmVersion': 'constantinople'
-        }
-    };
-}
-
-function getImports(dependency) {
-    console.log('Searching for dependency: ', dependency);
-}
-
-function compileSources(config) {
-    try {
-        return JSON.parse(solc.compile(JSON.stringify(config), getImports));
-    } catch (e) {
-        console.log(e);
-    }
-}
-
-function errorHandling(compiledSources) {
-    if (!compiledSources) {
-        console.error('>>>>>>>>>>>>>>>>>>>>>>>> ERRORS <<<<<<<<<<<<<<<<<<<<<<<<\n', 'NO OUTPUT');
-    } else if (compiledSources.errors) { // something went wrong.
-        console.error('>>>>>>>>>>>>>>>>>>>>>>>> ERRORS <<<<<<<<<<<<<<<<<<<<<<<<\n');
-        compiledSources.errors.map(error => console.log(error.formattedMessage));
-    }
-}
-
-module.exports.compileContracts = main;
-
-async function main () {
-    compilingPreperations();
-    const config = createConfiguration();
-    // console.log('config')
-    // console.log(config)
-    let output = compileSources(config);
-    errorHandling(output);
-    // console.log(output.contracts['GasHelper.sol']);
-    // console.log(output);
-
-    return output;
-}
-
-main();
```

Comment:
1 more thing I added here which I found helpful.
if contract is called with no script it just compiles. and better if it also output result.
so if anyone updates the compile contracts file. he can run it without needing any script for checking the update is working as expected.
So I suggest each compile contracts file will have 1 export function
and main function that calls the compile function and prints output.errors.